portfolio/src/main/webapp/index.html

```diff
@@ -2,13 +2,14 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Gábor Fuisz - 2020 STEP Internship Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1>Gábor Fuisz</h1>
+      <img src="images/selfie.jpg" id="selfie">
```

Comment:
I think you're missing the actual photo. You might have to add the new folder and the image inside manually via `git add images/selfie.jpg` so they get included in the commit.

Reply:
I'll approve this for simplicity of merging since you have the photo in the next PR. Usually you should have put the photo in this PR to make the change in itself complete. You could do this by checking out the branch again and adding the photo in a new commit.